lib/matplotlib/axes/_axes.py

```diff
@@ -3889,17 +3889,25 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
         ----------------
         kwargs : `~matplotlib.collections.Collection` properties
 
+        See Also
+        --------
+        plot : to plot scatter plots when markers are identical in size and
```

Comment:
No it doesn't, thought it is currently impossible to check on matplotlib's documentation.